projects/1.18/assets/extreme-sound-muffler/extremesoundmuffler/lang/zh_cn.json

```
@@ -0,0 +1,57 @@
+{
+    "key.open_muffler_gui": "打开消音器界面",
+  
+    "inventory.btn": "消音器",
+  
+    "slider.btn.muffler.muffle": "开启消音",
+    "slider.btn.muffler.unmuffle": "关闭消音",
+    "slider.btn.play.play_sound": "播放/暂停音效",
+    "slider.btn.volume": "音量: ",
+  
+    "main_screen.btn.csl.recent": "最近",
+    "main_screen.btn.csl.all": "所有",
+    "main_screen.btn.csl.muffled": "已消音",
+    "main_screen.btn.csl.tooltip": "显示 %s 音效",
+    "main_screen.btn.tms.stop": "停止消音",
+    "main_screen.btn.tms.start": "开始消音",
+    "main_screen.btn.delete.sounds": "删除消音音效",
+    "main_screen.btn.delete.list": "删除消音列表",
+    "main_screen.btn.delete.anchor": "删除锚点",
+    "main_screen.btn.accept": "确定",
+    "main_screen.btn.cancel": "关闭",
+    "main_screen.btn.next_sounds": "下一个音效",
+    "main_screen.btn.previous_sounds": "上一个音效",
+    "main_screen.btn.anchors.disabled": "锚点已禁用",
+    "main_screen.btn.anchors.set_message": "首先设置锚点",

```

| **审查级别**    | 备注 |
|--------------|----|
| ⚪ 轻微 / 不阻止合并但需三思  |    |

让它看起来像个祈使句
```suggestion
    "main_screen.btn.anchors.set_message": "请先设置锚点",
```
原文：**Set the anchor first**